src/API/Common/Export/Headers.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\API\Common\Export;
+
+class Headers
```

Comment:
It would be good to add description to this class, at least its purpose to avoid confusions